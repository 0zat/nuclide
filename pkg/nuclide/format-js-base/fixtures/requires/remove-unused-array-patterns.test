var x = require('x');

var [a, b] = require('foo');
var [e, f] = require('baz');
var [c, d] = require('bar');

e(f);
c(d);
x();
