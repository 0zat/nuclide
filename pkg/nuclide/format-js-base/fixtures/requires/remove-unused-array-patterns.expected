var [c, d] = require('bar');
var [e, f] = require('baz');
var x = require('x');

e(f);
c(d);
x();
